security/security-core/src/main/java/io/camunda/security/reader/ResourceAccessChecks.java

```diff
@@ -29,6 +29,7 @@ public List<String> getAuthorizedResourceIds() {
 
     return Optional.of(authorizationCheck)
         .map(AuthorizationCheck::authorization)
+        .map(List::getFirst)
```

Comment:
* This was necessary to keep the RDBMS implementation working => but that way, it only applies authorization checks for the first specified authorization in the list (in other words, it does not support disjunctive authorization checks)
* A better approach would be to remove this method actually and adjust the places where it has been used => The Authorization Check would already contain the mapping of "authorization" to a list of "resource ids" (and later properties as well)